modules/rest-api/pages/rest-fts-indexing.adoc

```diff
@@ -157,9 +157,10 @@ Supported index types include:
  ** bleve: a full-text index powered by the bleve engine
 
 * [.param]`planParams`: optional, string (JSON), form parameter
-* [.param]`prevIndexUUID`: optional, string, form parameter
+* [.param]`uuid`: optional, string, form parameter
 +
-Intended for clients that want to check that they are not overwriting the index definition updates of concurrent clients.
+uuid field is mandatory for index definition updates.
+It also helps the clients that want to check that they are not overwriting the index definition updates of other concurrent clients.
```

Comment:
Suggest rewording to:
"It also helps clients check that they do not overwrite the index definition updates of other concurrent clients"

Reply:
@amarantha-k ,  I am closing this PR and will revise this later.

Reply:
Hey @steveyen ,  as the prevIndexUUID option is supposed to be a query parameter, I think its better to change the document description this way by mentioning only about the `uuid` in the POST request form than bringing back the the `prevIndexUUID` into the URL as a query parameter?

Reply:
@amarantha-k, reopening this PR and will proceed as we sync with Steve on this.